src/main/scala/com/amazon/deequ/profiles/ColumnProfilerRunBuilder.scala

```diff
@@ -39,11 +39,17 @@ class ColumnProfilerRunBuilder(val data: DataFrame) {
   protected var saveColumnProfilesJsonPath: Option[String] = None
   protected var saveConstraintSuggestionsJsonPath: Option[String] = None
   protected var saveEvaluationResultsJsonPath: Option[String] = None
-  protected var correlation = true
-  protected var histogram = true
+  protected var correlation = false
+  protected var histogram = false
   protected var kllProfiling = false
-  protected var kllParameters: Option[KLLParameters] = None
+  protected var kllParameters: Option[KLLParameters] = Some(KLLParameters(2048,
+                                                                          0.64,
+                                                                          20))
```

Comment:
Those are the defaults set in `com.amazon.deequ.analyzers.KLLSketch` - except for the 20 (number of buckets), which is usually 10. However, I remove the default values here, as this will always be set by the hsfs client.